src/commonControls/numberControl/NumberControl.ts

```diff
@@ -110,17 +116,16 @@ export interface NumberControlProps extends ControlProps {
      *  - the Control will take initiative to explicitly confirm the value with a yes/no
      *    question.
      */
-    confirmationRequired?: boolean | NumberConfirmationRequireFunction;
+    confirmationRequired?: boolean | ((state: NumberControlState, input: ControlInput) => boolean);
```

Comment:
It will be common to want `confirmationRequired=true` whenever a value has a most-likely misunderstanding.  It would be great if there is a built-in function that provides this.

Reply:
Yes, created a new namespace with few default implementations
e.g `NumberControlBuiltIns.confirmMostLikelyMisunderstandingInputs`